widgets.md

```diff
@@ -14,10 +14,22 @@ extra_css:
 
 This page describes some of the widgets available in Flutter.
 
-See also:
-
-* [The API reference for the `widgets` library](https://docs.flutter.io/flutter/widgets/widgets-library.html).
-* [The API reference for the `material` library](https://docs.flutter.io/flutter/material/material-library.html).
+Flutter uses a layered approach. At the highest level we provide
+[Material components](#material-design-widgets). These are implemented
+in the [`material`
+library](https://docs.flutter.io/flutter/material/material-library.html).
+That library is implemented in terms of the [`widgets`
```

Comment:
I'm generally in favor of not using 'that', and instead being specific. "The material library is implemented.."